.ci/scripts/lint-commits.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
+set -xeo pipefail
 
-set -xe
+export PATH=${PATH}:/$(pwd)/node_modules:/$(pwd)/node_modules/.bin
+export HOME=$(pwd)
+
+npm install @commitlint/cli
```

Comment:
This should already be installed as a dev-dependency when running `npm install` - or is this because we're not installing the normal modules as part of this stage?

Reply:
I was trying to make it quicker, I mean not have to download all dependencies

Reply:
Good idea. Just wasn't sure if you also ran `npm install`. But if you don't then this is better 👍